### ISSUE TITLE
Sorting with <a> tags

### ISSUE THREAD

**Issue by flyinmryan** [tabalinas/jsgrid#245]
I am trying to get column sorting working when I have link tags in the columns.  Instead of sorting by the text of the link, it wants to sort on all the HTML `<a href=".....">` tags.

I went into sortStrategies and this is what I have come up with based on the other sorting functions:

`link: function (l1, l2) {`
     `       var link1 = l1.substring(l1.indexOf(">") + 1);`
     `       var link2 = l2.substring(l2.indexOf(">") + 1);`
     `       return link1 - link2;`
`}`

The variables link1 and link2 create proper substrings to sort on, but it doesn't seem to be sorting properly.  Have you seen something like this done before, or maybe know a better way to do it?

Thanks for your help.

**Comment by tabalinas** [comment]
What kind of data is coming from the back-end? Is it markup?
The data coming from server is used for sorting. So if it's an HTML coming from server it will be used by sorters.

I guess the better idea would be to provide just data to the grid and render the link with `itemTemplate`, so you won't have issues with sorting.
As for your current sorting function, I guess it's incorrect string sorting. See how sorting for string is defined in the sources https://github.com/tabalinas/jsgrid/blob/master/src/jsgrid.sort-strategies.js

**Comment by flyinmryan** [comment]
Ok, I've gotten it to work!  I wasn't sure what the localeCompare function did, but I tried it and it works.  Here the is final function I added to sortStrategies:

```javascript
link: function (l1, l2) {
            var link1 = l1.substring(l1.indexOf(">") + 1);
            var link2 = l2.substring(l2.indexOf(">") + 1);
            return ("" + link1).localeCompare("" + link2);
        }
```

You asked what data was coming form the back end:  I'm building the table data client side.  I make an AJAX call and build an array of objects, and pass the array in the data parameter when initializing the grid.

Thanks for your help as always
